pkg/smartcontract/parameter_test.go

```diff
@@ -122,17 +122,20 @@ var marshalJSONTestCases = []struct {
 		},
 		result: `{"type":"Hash256","value":"0xf037308fa0ab18155bccfc08485468c112409ea5064595699e98c545f245f32d"}`,
 	},
+	{
+		input: Parameter{
+			Type:  InteropInterfaceType,
+			Value: nil,
+		},
+		result: `{"type":"InteropInterface","value":"null"}`,
```

Comment:
If this is intended to be JSON's `null`, then quotes aren't needed.

Reply:
ACK